Microsoft.Toolkit.Uwp/Helpers/PrintHelper/PrintHelper.cs

```diff
@@ -373,7 +372,7 @@ private void ApplyPrintSettings(PrintTask printTask)
         /// </summary>
         /// <param name="sender">PrintDocument</param>
         /// <param name="e">Paginate Event Arguments</param>
-        private void CreatePrintPreviewPages(object sender, PaginateEventArgs e)
+        private async void CreatePrintPreviewPages(object sender, PaginateEventArgs e)
```

Comment:
Should this be an async Task (instead of async void)?

Reply:
That method is an event handler, which is the exception for the `async Task` rule. More info [here](https://msdn.microsoft.com/en-us/magazine/jj991977.aspx)